.github/workflows/sync-packages.yml

```
@@ -0,0 +1,38 @@
+name: Sync packages
+
+on:
+  schedule:
+    - cron: '0 0 * * 0' # At 00:00 on Sunday.
+  workflow_dispatch: # Allow manual triggering
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: pnpm/action-setup@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22
+          cache: pnpm
+      - run: pnpm install --frozen-lockfile
```

```suggestion
      - run: pnpm install --frozen-lockfile --ignore-scripts
```

don't run any scripts to make it less likely that something can corrupt the repo, esp with contents: write permission